docs/layers/software-delivery/eks-argocd/setup.mdx

```diff
@@ -12,118 +12,218 @@ import StepNumber from '@site/src/components/StepNumber'
-    PAT is given the least-access required for the given job.
-
-    Each one of these PATs will be associated with a given user. We recommend creating or using an existing "bot" user. For
-    example, at Cloud Posse we have the "cloudpossebot" GitHub user. This user has its own email address and GitHub account,
-    is accessible from our internal 1Password vault for all privileged users, and has all access keys and tokens stored with
-    it in 1Password.
-
-    This bot user will need permission to manage a few repositories in your Organization. If you wish to simplify
-    deployment, you can grant this user permission to create repositories. See (Create Permission for the Bot User)[#Create
-    Permission for the Bot User].
-
-    Use this bot user for all access tokens in the remainder of this guide.
-  </Step>
-
-  <Step>
-    ## <StepNumber/> Create ArgoCD GitHub Repositories
+    ## <StepNumber/> Create Argo CD GitHub Repositories
 
     <Steps>
-      - Create the two required ArgoCD GitHub repos:
+      - Create the two required Argo CD GitHub repos:
         <TaskList>
         - [acme/argocd-deploy-non-prod](https://github.com/acme/argocd-deploy-non-prod)
         - [acme/argocd-deploy-prod](https://github.com/acme/argocd-deploy-prod)
         </TaskList>
-
-      - Then grant the bot user `Admin` access to these two repositories.
     </Steps>
   </Step>
 
   <Step>
-    ## <StepNumber/> Create all ArgoCD PATs
+    ## <StepNumber/> Prepare Authentication
+
+    Argo CD can be integrated with GitHub using either GitHub Apps (recommended) or Personal Access Tokens (PATs). GitHub Apps provide more granular permissions, better security, and improved audit capabilities.
+
+    Argo CD requires several different types of GitHub authentication for various components and workflows. While these could be combined, we follow the principle of least privilege by creating separate authentication credentials for each specific purpose. The following authentication methods are required:
+
+    <Steps>
+    1. #### Terraform `argocd-repo` Access
+       The first PAT is used by Terraform with the `argocd-repo` component. This is used to create and manage the Argo CD desired state repositories and (optionally) add deployment ssh keys for those repos.
 
-    We will need several PATs for the following steps. Now that the ArgoCD repos are created, you can create all required
-    PATs.
+       Terraform will pull that PAT from SSM typically using the `argocd/github` path in the `core-auto` account.
 
-    Please see
-    [ArgoCD Integrations: How to set up Authorization for ArgoCD with GitHub PATs](/layers/software-delivery/eks-argocd/tutorials/pats)
-    and follow all steps.
+       Alternatively, you can required local access to apply the component. This would require an engineer to locally authenticate with GitHub and apply this component locally. Since this component is rarely updated, this can be a reasonable trade-off.
```

Comment:
```suggestion
       Alternatively, you can enforce local access to apply the component. This would require an engineer to locally authenticate with GitHub and apply this component locally. Since this component is rarely updated, this can be a reasonable trade-off.
```